cosmos/dbt/project.py

```diff
@@ -6,13 +6,16 @@
     DBT_LOG_DIR_NAME,
     DBT_TARGET_DIR_NAME,
 )
+from cosmos.config import RenderConfig
 from contextlib import contextmanager
 from typing import Generator
 
 
 def create_symlinks(project_path: Path, tmp_dir: Path) -> None:
     """Helper function to create symlinks to the dbt project files."""
-    ignore_paths = (DBT_LOG_DIR_NAME, DBT_TARGET_DIR_NAME, "dbt_packages", "profiles.yml")
+    ignore_paths = [DBT_LOG_DIR_NAME, DBT_TARGET_DIR_NAME, "profiles.yml"]
+    if RenderConfig.dbt_deps is False:
+        ignore_paths.append("dbt_packages")
```

Comment:
@DanMawdsleyBA confirming our intended behaviour:

1. when the user specifies `dbt_deps == True`, since Cosmos installs the deps, we don't want to create the symlink to `dbt_packages`
2. if the user sets  `dbt_deps == False`, we assume the deps are pre-installed, so we want to create symlinks

Is this the case?

Reply:
@DanMawdsleyBA I believe the integration test is failing because your check here is on the class `RenderConfig` instead of the `render_config` object. `RenderConfig.dbt_deps` will always be `True` because that's the default [here](https://github.com/DanMawdsleyBA/astronomer-cosmos/blob/049d99c01483b0689f145c1e7be00e01f62ce439/cosmos/config.py#L54). 

It might make more sense in `create_symlinks` to have an arg for `ignore_dbt_packages` or something like that when it's called here: https://github.com/astronomer/astronomer-cosmos/blob/e1f34ea424b8940f1c06660554298008876ea11b/cosmos/dbt/graph.py#L235

Reply:
@jbandoro that's a good spot thanks! I have updated it to pass through the actual dbt deps. I get your point on the name and it is a little confusing but wonder if adding an additional variable is necessary as the value of dbt deps is directly linked to whether we want to link the packages folder or not. Open to counter suggestion and thoughts @tatiana

Reply:
Thanks, @jbandoro and @DanMawdsleyBA, for following up on this!

I liked the suggestion. What if we renamed the argument from `dbt_deps` to `skip_dbt_packages` (only within `create_symlinks`)? From the `create_symlinks` function, this name would probably make more sense. I'm happy with `ignore_dbt_packages` as well.

The name `dbt_deps` continues to make sense in the context of `load_via_dbt_ls`.

Reply:
Yeah that's a good middle ground I updated the name to ignore_dbt_packages (to align it with the ignore paths name) . I added in a comment for future reference